donate/templates/pages/core/contributor_support_page.html

```diff
@@ -0,0 +1,87 @@
+{% extends "pages/base_page.html" %}
+{% load form_tags static wagtailcore_tags wagtailimages_tags i18n %}
+
+{% block content %}
+
+<header class="page-header">
+    <div class="page-header__container">
+        <h1 class="page-header__heading">{{ page.title }}</h1>
+    </div>
+</header>
+
+<div class="layout">
+    <div class="layout__container">
+        <div class="layout__full-col">
+            <div class="page-body">
+                <form class="support-form action="https://mozillasalescloud--mkl.cs4.my.salesforce.com/servlet/servlet.WebToCase?encoding=UTF-8" method="POST">
+                    <p class="rich-text">{% blocktrans %}
+                        Here is some intro guide text to explain what this form is for and who should be filling it out.
+                    {% endblocktrans %}</p>
+
+                    <input type=hidden name="orgid" value="{{ orgid }}">
+
+                    {# TODO: should the retURL be a static "thank you for your question" page rather than "the current page"? #}
+                    <input type=hidden name="retURL" value="{{ page.full_url }}">
```

Comment:
This is an open question... @alanmoo @stephaniemcv

Reply:
I have no opinion/idea. Do whatever's easy.